File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        l=len(nums)
        for i in range(l):
            for j in range(i+1,l):
                if nums[i]+nums[j]==target:
                    return [i,j]
                    break
        
        return
        
        

File: 0001-two-sum/0001-two-sum.go
func twoSum(nums []int, target int) []int {
 
    seen := make(map[int]int, len(nums))
    
    for currentIndex, num := range nums {
        complement := target - num
        
  
        if mapIndex, found := seen[complement]; found {
            return []int{mapIndex, currentIndex}
        }
        
        seen[num] = currentIndex
    }
    
    return nil
}

